packages/edit-post/src/store/actions.js

```diff
@@ -147,10 +148,13 @@ export function removeEditorPanel( panelName ) {
  * @param {string} feature Feature name.
  */
 export const toggleFeature = ( feature ) => ( { registry } ) =>
-	registry
-		.dispatch( interfaceStore )
-		.toggleFeature( 'core/edit-post', feature );
+	registry.dispatch( preferencesStore ).toggle( 'core/edit-post', feature );
```

Comment:
There is the option of deprecating the existing actions and selectors in `edit-post`.

There's quite a lot of usage across the codebase (and possibly in plugins too), so I haven't done that. I'd be happy to address that in a follow-up if reviewers feel a deprecation is best.

Reply:
Yes I think we should deprecate it. Otherwise future devs will mistakenly use this one. Since it is a method that made it into Core, we can't _remove_ the method, but that's okay.

Reply:
What form could that deprecation take?

I'm thinking we could add some sort of message in the actions and selectors but we might never be able to remove them altogether without risk of breaking things for plugins. It would be good to update usage in this codebase anyway, to make sure we're setting the right example (definitely a follow-up job though!)

Reply:
Yes because it shipped in Core we have to follow the Core deprecation policy which is to call `deprecated()` (or any of the `deprecated_*` methods in PHP) but never actually remove the method. It will spit out a a warning in the console but continue working.